restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -969,7 +971,8 @@ private void updateExistenceForMap(
    * @throws ExecutionException
    * @throws InterruptedException
    */
-  private ImmutablePair<List<Row>, ByteBuffer> searchRows(
+  @VisibleForTesting
```

Comment:
this change made the codancy fail with the error:
`Avoid really long methods.`

Reply:
I did not change the method body, just made it more accessible so that tests could use simple mocks for this method (previously PowerMock was required, but it does not work well with JUnit 5).

Perhaps Codacity flags this error because the method declaration changed... :thinking: Is it worth fixing at this point? The purpose of this PR was mainly to re-enable the unit tests that we not being executed. I wanted to minimize changes to production code in this PR. WDYT?

Reply:
This problem is already reported on `master` so this PR do not make the situation worse :) https://app.codacy.com/gh/stargate/stargate/file/51191973112/issues/source?bid=21416080&fileBranchId=21416080

I had a go at factoring some of the code out, but it is not trivial as there pieces of the code inside this method are interdependent logically. So in effect this method is indeed complex and some simplification would be good, but at the same time I'd like to avoid scope creep on this PR. I'm ok with opening an follow-up issue for this though. WDYT?

Reply:
good idea with the follow-up ticket.

Reply:
#537